galpy/potential/TwoPowerSphericalPotential.py

```diff
@@ -181,8 +315,143 @@ def _dens(self,R,z,phi=0.,t=0.):
         HISTORY:
            2010-08-08 - Written - Bovy (NYU)
         """
-        r= numpy.sqrt(R**2.+z**2.)
-        return (self.a/r)**self.alpha/(1.+r/self.a)**(self.beta-self.alpha)/4./m.pi/self.a**3.
+        a, alpha, beta = self.a, self._alpha, self._beta
+        r= sqrt(R**2.+z**2.)
+        return (a/r)**alpha/(1.+r/a)**(beta-alpha)/4./m.pi/a**3.
+
+    def _mass(self,R,z=0.,t=0.):
+        """
+        NAME:
+           _mass
+        PURPOSE:
+           evaluate the mass within R for this potential
+        INPUT:
+           R - Galactocentric cylindrical radius
+           z - vertical height
+           t - time
+        OUTPUT:
+           the mass enclosed
+        HISTORY:
+           2014-04-01 - Written - Erkal (IoA)
+        """
+        a, alpha, beta = self.a, self._alpha, self._beta
+        r= R if z is None else sqrt(R**2.+z**2.)
+        # if z is None:
+        #     r= R
+        # else:
+        #     r= sqrt(R**2.+z**2.)
+        return (r/a)**(3.-alpha)/(3.-alpha)*hyp2f1(3.-alpha,-alpha+beta,4.-alpha,-r/a)
```

Comment:
Unnecessary change?

Reply:
the `r= R if z is None else sqrt(R**2.+z**2.)` should be infinitesimally faster

Reply:
Ah, cool, then we should keep it; but remove the commented out lines

Reply:
done. and changed them all.